packages/stores/package.json

```diff
@@ -8,6 +8,7 @@
         "mobx": "^6.6.1",
         "mobx-persist-store": "1.1.2",
         "mobx-react-lite": "^3.4.0",
+        "moment": "^2.29.2",
```

Comment:
@shafin-deriv Please move it to `devDependencies` 🙏🏻

Reply:
Aha! Sorry about that!